### DISCUSSION TITLE
Submission error'异常'

### DISCUSSION THREAD

**Discussion by 9898zy998** [PaddlePaddle/PaddleSpatial#181] [BaiduKDDCup2022]
When I submit my code, the status of submission is 'Error occurred' (异常).

My structure:
paddlepaddle.zips
| --- checkpoints/
             | --- models/
                          | ---model_0
                          | --- ...
| ---predict.py
| ---prepare.py
| ---common.py
| ---model.py
| ---train.py
| ---wind_turbine_data.py
Is there something wrong in this sturcture?

Thank you in advance.

**Comment by 9898zy998** [comment]
Well,the structure is like:
<img width="246" alt="image" src="https://user-images.githubusercontent.com/72608995/174099217-fd9d5c15-97fb-4b3d-a882-e8b22c7fdd6b.png">
My team name is ‘123123’. We get the lastest error message in 2022-06-16 22:20.

**Comment by zhiyansong** [comment]
Hello, I have encountered the same problem, may I ask whether your problem has been solved?
